vue/src/components/editor/MapUtils.js

```diff
@@ -0,0 +1,78 @@
+import { Vector3 } from 'three'
+import { Map } from './Map'
```

Comment:
I don't think so. I just needed them for the javadoc style comments, otherwise the @param {Vector} wouldn't be type defined when you look at the functions.  I'll take this out though.